resizable/src/main/scala/lucuma/react/resizable/Resizable.scala

```diff
@@ -48,8 +48,11 @@ final case class Resizable(
 
 object Resizable {
   @js.native
-  @JSImport("react-resizable", "Resizable")
-  object RawComponent extends js.Object
+  @JSImport("react-resizable", JSImport.Default)
```

Comment:
Cool, I tried some combinations like this but couldn't find one that would workk